engine/src/main/java/org/destinationsol/game/SolCam.java

```diff
@@ -110,6 +117,8 @@ public void updateMap(SolGame game) {
         myZoom = SolMath.approach(myZoom, desiredZoom, ZOOM_CHG_SPD * ts);
         applyZoom(game.getMapDrawer());
         myCam.update();
+        viewport.update(Gdx.graphics.getWidth(), -Gdx.graphics.getHeight());
```

Comment:
Use displayDimensions here (and in the next line) instead of using Gdx.graphics.whatever

Reply:
Fixed, I think, with #321